lib/toxiproxy_ex.ex

```diff
@@ -1,11 +1,9 @@
 defmodule ToxiproxyEx do
-  alias ToxiproxyEx.{Proxy, Client, Toxic, ToxicCollection, ServerError}
+  @moduledoc """
```

Comment:
I prefer to keep the README parsing.

Reply:
In the https://hexdocs.pm/toxiproxy_ex/api-reference.html, the first paragraph for each Elixir module will be the short description for a module. If we want to add badges, to prevent badges from being parsed as first paragraph, the best approach to set `readme` as default page.

Reply:
Alright got it! I didn't notice that hexdoc will show a truncated description there.

I would propose to either add a new `<!-- MDOC !-->` comment to not include the batches or add a useful standalone comment with some information.

Reply:
I changed the README and moduledoc to have a simpler tagline on `main` already: 
![image](https://user-images.githubusercontent.com/3436381/96547369-436fb000-12ac-11eb-987a-b559fcc079f4.png)

Maybe we can now just add the badges above the first `<!-- MDOC !-->` so that it is not included in the moduledoc.

Reply:
> I changed the README and moduledoc to have a simpler tagline on `main` already:
> ![image](https://user-images.githubusercontent.com/3436381/96547369-436fb000-12ac-11eb-987a-b559fcc079f4.png)
> 
> Maybe we can now just add the badges above the first `<!-- MDOC !-->` so that it is not included in the moduledoc.

Noted. FYI, these badges will not appear in the HexDocs.